<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QLineEdit" name="reg_FIO">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>ФИО</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="reg_works">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="reg_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="reg_password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registration_reg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
